docs/application/dotnet/getting-started/sec-privileges.md

```diff
@@ -0,0 +1,191 @@
+| `http://tizen.org/privilege/email` | public |  | The application can manage user's email accounts, including user's folders and emails, POP3 and IMAP downloads, and SMTP uploads. This may result in additional charges depending on user's payment plan. |
+| `http://tizen.org/privilege/email.admin` | platform |  | The application can manage the settings of email applications. |
+| `http://tizen.org/privilege/externalstorage` | public | Storage | The application can read and write files that are saved to external storage, such as SD cards. |
+| `http://tizen.org/privilege/externalstorage.appdata` | public |  | The application can read and write its own files in external storage, such as SD cards. |
+| `http://tizen.org/privilege/fido.client` | public |  | The application can trigger authenticators in user's device and it may request to use user's PIN or biometrics (fingerprints or irises) for authentication. |
+| `http://tizen.org/privilege/gestureactivation` | platform |  | The application can allow and block special touch gestures. |
+| `http://tizen.org/privilege/gesturegrab` | platform |  | The application can read special touch gestures, even while it is running in the background. |
+| `http://tizen.org/privilege/haptic` | public |  | The application can control vibration feedback. |
+| `http://tizen.org/privilege/healthinfo` | public | Sensor | The application can read health information gathered by the device sensors, such as the pedometer and the heart rate monitor. |
+| `http://tizen.org/privilege/ime` | public |  | The application can provide users with a way to enter characters and symbols into an associated text field. |
+| `http://tizen.org/privilege/imemanager` | public |  | The application can manage installed input methods. |
+| `http://tizen.org/privilege/inputgenerator` | platform |  | The application can simulate keys being pressed and touch interactions with the screen. |
+| `http://tizen.org/privilege/internet` | public |  | The application can access the Internet. This may result in additional charges depending on user's payment plan. |
+| `http://tizen.org/privilege/keygrab` | platform |  | The application can read actions involving special keys, such as the volume keys on this or other devices (e.g. TV remote controls), even when it is running in the background. |
+| `http://tizen.org/privilege/keymanager` | public |  | The application can save keys, certificates, and data to, and retrieve and delete them from, password-protected storage. Checking the statuses of certificates while connected to a mobile network may result in additional charges depending on user's payment plan. |
+| `http://tizen.org/privilege/keymanager.admin` | platform |  | The application can lock and unlock password-protected storage, and manage password changes for it. |
+| `http://tizen.org/privilege/led` | public |  | The application can turn LEDs on or off, such as the LED on the front of the device and the camera flash. |
```

Comment:
The application can turn LEDs on or off. For example, the LED found on the front of the device and the camera flash can be turned on or off.

Reply:
done